schema/dataKeys.json

```diff
@@ -115,7 +115,7 @@
       "type": "object",
       "properties": {
         "product_id": {
-          "type": ["array", "integer"],
+          "type": ["array"],
```

Comment:
I think it should not have the brackets anymore, just `"type": "array",`

Reply:
Updated, thanks